### ISSUE TITLE
Please add additional explanation

### ISSUE THREAD

**Issue by jefftrull** [abeMedia/jekyll-pdf#15]
The configuration is explained (Gemfile and _config.yml modifications) but not how to actually cause the PDFs to be generated.  Does it happen at jekyll build time? Or when the site is served? Where is the file stored?  I have made the specified changes, but cannot find any sign that a PDF has been generated. Thank you.

**Comment by abemedia** [comment]
Good point. The PDFs are generated at build time. I have just realised an issue when upgrading to the latest version of `jekyll-assets` though so could be that. I will look into it this weekend.

**Comment by hoijui** [comment]
see https://github.com/abeMedia/jekyll-pdf/issues/24
